src/style.css

```diff
@@ -1,3 +1,56 @@
-body {
+html {
+  font-family: Roboto, serif;
```

Comment:
Consider using a web-safe font stack or including a fallback font in case 'Roboto' is not available. This ensures better cross-browser compatibility.